tests/wpunit/Tribe/Events/Models/Event_ModelTest.php

```diff
@@ -46,6 +48,28 @@ public function it_should_return_past() {
 		$this->assertTrue( $event->is_past );
 	}
 
+	/**
+	 * @test
+	 */
+	public function it_should_return_is_now() {
```

Comment:
This test, and the following one, could be conflated in a `test_is_now_property` test that would be fed data from a data provider to create an Event happening at a certain offset, with respect to the current moment, and an expected `is_now` value.

E.g. `test_is_now_property( string $when, int $duration, bool $expected ) : void`.